src/MiNET/MiNET.Test/GenerateItemsTests.cs

```diff
@@ -144,6 +145,14 @@ private GenerationItemInfo GetGenInfoByKnownItemIds(string id, Dictionary<string
 			{
 				return new GenerationItemInfo(typeof(ItemCarpetBase).Name);
 			}
+			if (id.EndsWith("hanging_sign"))
+			{
+				return new GenerationItemInfo(typeof(ItemHangingSignBase).Name);
+			}
+			if (id.EndsWith("standing_sign"))
```

Comment:
this is system Item for StandingSign block only. Think gen isn't necessary for it